### PR TITLE
chore: add lab flag in eigen to enable inquiry checkout

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-2332]

Pretty straightforward - just adding a lab flag so work can begin on inquiry checkout

[PURCHASE-2332]: https://artsyproduct.atlassian.net/browse/PURCHASE-2332

### CODE REVIEW

**Review by zephraph — APPROVED** [review]
Looks good, thanks @lilyfromseattle!

**Review by erikdstock — APPROVED** [review]

File: Artsy_Tests/App_Tests/ARAppDelegateEmissionSpec.m [hunk 1]
```diff
@@ -11,6 +11,7 @@
         @"AREnableBuyNowFlow": @(YES),
         @"AROptionsArtistSeries": @(NO),
         @"AROptionsNewFirstInquiry": @(NO)
```
Comment (zephraph) [inline Artsy_Tests/App_Tests/ARAppDelegateEmissionSpec.m:13]:
```suggestion
        @"AROptionsNewFirstInquiry": @(NO),
```

File: Artsy_Tests/App_Tests/ARAppDelegateEmissionSpec.m [hunk 2]
```diff
@@ -30,6 +31,7 @@
       @"AROptionsPriceTransparency": @(NO),
       @"AROptionsArtistSeries": @(NO),
       @"AROptionsNewFirstInquiry": @(NO)
```
Comment (zephraph) [inline Artsy_Tests/App_Tests/ARAppDelegateEmissionSpec.m:33]:
```suggestion
      @"AROptionsNewFirstInquiry": @(NO),
```

**Review by zephraph — CHANGES_REQUESTED** [review]
Missed some commas that caused the build to fail

File: Artsy/App/AROptions.m [hunk 3]
```diff
@@ -46,6 +47,7 @@ + (void)initialize
          AROptionsLoadingScreenAlpha: @"Loading screens are transparent",
          AROptionsUseReactNativeWebView: @"Use react native webviews",
          AROptionsNewInsightsPage: @"Enable artist insights page",
+         AROptionsInquiryCheckout: @"AROptionsInquiryCheckout",
```
Comment (lilyfromseattle) [inline Artsy/App/AROptions.m:50]:
```suggestion
         AROptionsInquiryCheckout: @"Enable inquiry checkout",
```

**Review by zephraph — APPROVED** [review]

**Pull request merged**